app/controllers/admin/resets_controller.rb

```diff
@@ -6,7 +6,7 @@ class ResetsController < BaseController
 
     def create
       authorize @user, :reset_password?
-      @user.send_reset_password_instructions
+      @user.reset_password!
       log_action :reset_password, @user
       redirect_to admin_accounts_path
```

Comment:
This should probably redirect to the account whose passwords has been reset, instead of the list of accounts.